PowerValidatedSolutions.psm1

```diff
@@ -50066,6 +50066,372 @@ Function Test-WMSubnetInput {
 }
 Export-ModuleMember -Function Test-WMSubnetInput
 
+Function Invoke-GeneratePrivateKeyAndCsrFiles {
+    <#
+        .SYNOPSIS
+        Generate Private Key and certificate signing request (CSR) files using Openssl with a common name and neccesary parameters.
```

Comment:
```suggestion
        Generate the private Key and certificate signing request (CSR) files using Openssl with a common name additional parameters as necessary.
```

Reply:
addressed in commit 9d50a853ffce55094985dea3d9e658670a7ab67a